commands/1_clicks.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/spf13/cobra"
+)
+
+// OneClicks creates the 1-click command.
+func OneClicks() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "1-click",
+			Short: "Display commands that pertain to 1-click applications",
+			Long: `The commands under ` + "`" + `doctl 1-click` + "`" + ` are for interacting with DigitalOcean 1-Click applications.
+With the 1-click command, you can list 1-Click applications.`,
```

Comment:
```suggestion
			Long:  "The commands under `doctl 1-click` are for interacting with DigitalOcean 1-Click applications.",
```

One tiny nitpick: I'd remove the second line here.